rust-install/src/install.rs

```diff
@@ -16,16 +16,21 @@ use std::env;
 
 #[cfg(not(windows))]
 const REL_MANIFEST_DIR: &'static str = "lib/rustlib";
+
+// FIXME: (brson) I don't think this should be bin/ even though
+// windows puts its libraries in bin.  rust-installer uses lib/ no
+// matter what platform, so using bin here makes the metadata used by
+// multirust-rs incompatible with rust-installer.
```

Comment:
I'm happy for that to change, it's simply the case that currently, rust on windows places rustlib in `bin/`.